[error] Cockroach DB tests return "missing destination name table_name in *[]pop.table"

Hi!

I was running tests for cockroach db and got error on `TruncateAll` method:

```
level=debug msg="select table_name from information_schema.tables;"
--- FAIL: Test_ActionSuite (0.02s)
    --- FAIL: Test_ActionSuite/Test_Auth_Login (0.01s)
        Error Trace:    model.go:20
    	            	suite.go:47
    	            	suite.go:88
    	Error:      	Received unexpected error:
    	            	missing destination name table_name in *[]pop.table
    	            	github.com/danikarik/coke-demo/vendor/github.com/markbates/pop.genericSelectMany
...
```

To reproduce error you can clone this [repo](https://github.com/danikarik/coke-demo).

Steps:
1) install buffalo with `go get -u -v github.com/gobuffalo/buffalo/buffalo`
2) run `dep ensure`
3) execute `buffalo test`

--- a/cockroach.go
+++ b/cockroach.go
@@ -230,11 +230,11 @@ func (p *cockroach) LoadSchema(r io.Reader) error {
 
 func (p *cockroach) TruncateAll(tx *Connection) error {
 	type table struct {
-		TableName string `sql:"table_name"`
+		TableName string `db:"table_name"`
 	}
 
 	tables := []table{}
-	if err := tx.RawQuery("select table_name from information_schema.tables;").All(&tables); err != nil {
+	if err := tx.RawQuery("select table_name from information_schema.tables where table_schema = ?;", tx.Dialect.Details().Database).All(&tables); err != nil {
 		return err
 	}
 

--- a/executors_test.go
+++ b/executors_test.go
@@ -329,3 +329,30 @@ func Test_Destroy_UUID(t *testing.T) {
 		r.Equal(count, ctx)
 	})
 }
+
+func Test_TruncateAll(t *testing.T) {
+	count := int(0)
+	transaction(func(tx *pop.Connection) {
+		a := require.New(t)
+
+		var err error
+		count, err = tx.Count("users")
+		user := User{Name: nulls.NewString("Mark")}
+		err = tx.Create(&user)
+		a.NoError(err)
+		a.NotEqual(user.ID, 0)
+
+		ctx, err := tx.Count("users")
+		a.Equal(count+1, ctx)
+	})
+
+	transaction(func(tx *pop.Connection) {
+		a := require.New(t)
+
+		err := tx.TruncateAll()
+		a.NoError(err)
+
+		ctx, _ := tx.Count("users")
+		a.Equal(count, ctx)
+	})
+}

